src/vm/comutilnative.cpp

```diff
@@ -1524,7 +1528,11 @@ FCIMPL5(VOID, Buffer::InternalBlockCopy, ArrayBase *src, int srcOffset, ArrayBas
     _ASSERTE(count >= 0);
 
     // Copy the data.
+#if defined(_WIN64)  
```

Comment:
This should be `#if defined(_AMD64_) && !defined(PLATFORM_UNIX)`. Otherwise, there will be build break on ARM64 and Unix will get slower for no good reason.